### PR TITLE
[zh-cn]: Object proofread part 2

### PR DESCRIPTION
**Pull request by JasonLamv-t**
### Description

校对并修正一些小的错漏。按照 MDN 目录索引，本部份自 entries() 到 values()

### Motivation

Ditto.

### Additional details

None.

### Related issues and pull requests

#13211

### CODE REVIEW

File: files/zh-cn/web/javascript/reference/global_objects/object/freeze/index.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ Object.freeze(obj)
 
 ## 描述
 
-冻结一个对象相当于[阻止其扩展](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Object/preventExtensions)然后将所有现有属性的描述符的 `configurable` 特性更改为 `false`——对于数据属性，将同时把 `writable` 特性更改为 `false`。无法向被冻结的对象的属性中添加或删除任何内容。任何这样的尝试都将失败，可能是静默失败，也可能抛出一个 {{jsxref("TypeError")}} 异常（通常情况下，在{{jsxref("Strict_mode", "严格模式", "", 1)}}中抛出）。
+冻结一个对象相当于[阻止其扩展](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Object/preventExtensions)然后将所有现有[属性的描述符](/zh-CN/docs/Web/JavaScript/Reference/Global_Objects/Object/defineProperty#描述)的 `configurable` 特性更改为 `false`——对于数据属性，将同时把 `writable` 特性更改为 `false`。无法向被冻结的对象的属性中添加或删除任何内容。任何这样的尝试都将失败，可能是静默失败，也可能抛出一个 {{jsxref("TypeError")}} 异常（通常情况下，在{{jsxref("Strict_mode", "严格模式", "", 1)}}中抛出）。
```
Comment (yin1999) [inline files/zh-cn/web/javascript/reference/global_objects/object/freeze/index.md:31]: adding this without modify the upstream?

https://github.com/mdn/content/blob/a0f6bf6f7d148f368f6965255058df1ed1f43839/files/en-us/web/javascript/reference/global_objects/object/freeze/index.md?plain=1#L10

File: files/zh-cn/web/javascript/reference/global_objects/object/freeze/index.md [hunk 2]
```diff
@@ -165,7 +165,7 @@ employee.address.city = "Noida"; // 可以修改子对象的属性
 console.log(employee.address.city); // "Noida"
 ```
 
-为了使对象不可变，需要递归地冻结每个对象类型的属性（深冻结）。当且仅当你知道对象在引用图中不包含[循环引用](<https://zh.wikipedia.org/wiki/環_(圖論)>)时，可以根据你的设计逐个案例地使用该模式，否则会导致无限循环。`deepFreeze()` 的一个增强功能是添加一个内部函数，该函数接收一个路径（例如一个数组）参数，这样可以在调用 `deepFreeze()` 递归地冻结对象时，避免冻结正在被处理的对象。但仍有可能会冻结不应该被冻结的对象，例如 \[window]。
+为了使对象不可变，需要递归地冻结每个对象类型的属性（深冻结）。当且仅当你知道对象在引用图中不包含[循环引用](<https://zh.wikipedia.org/wiki/環_(圖論)>)时，可以根据你的设计逐个案例地使用该模式，否则会导致无限循环。`deepFreeze()` 的一个增强功能是添加一个内部函数，该函数接收一个路径（例如一个数组）参数，这样可以在调用 `deepFreeze()` 递归地冻结对象时，避免冻结正在被处理的对象。但仍有可能会冻结不应该被冻结的对象，例如 [`window`](/zh-CN/docs/Web/API/Window)。
```
Comment (yin1999) [inline files/zh-cn/web/javascript/reference/global_objects/object/freeze/index.md:168]: same.

**Review by yin1999 — COMMENTED** [review]
Hi @JasonLamv-t, could you have a check with the english documents?

**Comment by JasonLamv-t** [comment]
> Hi @JasonLamv-t, could you have a check with the english documents?

Sorry about that, I have already checked the English version simultaneously, but I forgot to create a PR to the content repo because I'm kinda busy this week. I am going to turn this PR into a draft and will turn it back when the upstream content was updated.

File: files/zh-cn/web/javascript/reference/global_objects/object/freeze/index.md (hunk 1 above)
Comment (JasonLamv-t) [reply files/zh-cn/web/javascript/reference/global_objects/object/freeze/index.md:31]: This line already has a link to the properties' descriptor in the content repo's main branch, see https://github.com/mdn/content/blob/bcfc519e56326977d6a5d6bae279393441219b1c/files/en-us/web/javascript/reference/global_objects/object/freeze/index.md?plain=1#L33

File: files/zh-cn/web/javascript/reference/global_objects/object/freeze/index.md (hunk 2 above)
Comment (JasonLamv-t) [reply files/zh-cn/web/javascript/reference/global_objects/object/freeze/index.md:168]: Updated in https://github.com/mdn/content/pull/27286.

File: files/zh-cn/web/javascript/reference/global_objects/object/hasown/index.md [hunk 3]
```diff
@@ -32,26 +32,26 @@ Object.hasOwn(obj, prop)
 
 如果指定的属性是该对象的直接属性——**`Object.hasOwn()`** 方法返回 `true`，即使属性值是 `null` 或 `undefined`。如果属性是继承的或者不存在，该方法返回 `false`。它不像 {{jsxref("Operators/in", "in")}} 运算符，这个方法不检查对象的原型链中的指定属性。
 
-建议使用此方法替代 {{jsxref("Object.hasOwnProperty()")}}，因为它适用于使用 `Object.create(null)` 创建的对象以及覆盖了继承的 `hasOwnProperty()` 方法的对象。尽管可以通过在外部对象上调用 `Object.prototype.hasOwnProperty()` 解决这些问题，但是 `Object.hasOwn()` 更加直观。
+建议使用此方法替代 {{jsxref("Object.hasOwnProperty()")}}，因为它适用于使用 `Object.create(null)` 创建的对象以及重写了继承的 `hasOwnProperty()` 方法的对象。尽管可以通过在外部对象上调用 `Object.prototype.hasOwnProperty()` 解决这些问题，但是 `Object.hasOwn()` 更加直观。
```
Comment (yin1999) [inline files/zh-cn/web/javascript/reference/global_objects/object/hasown/index.md:35]:
```suggestion
建议使用此方法替代 {{jsxref("Object.hasOwnProperty()")}}，因为它适用于使用 `Object.create(null)` 创建的对象且重写了继承的 `hasOwnProperty()` 方法的对象。尽管可以通过在外部对象上调用 `Object.prototype.hasOwnProperty()` 解决这些问题，但是 `Object.hasOwn()` 更加直观。
```

**Review by yin1999 — COMMENTED** [review]
Just a suggestion

**Review by yin1999 — APPROVED** [review]
Thank you :+1:

**Pull request merged**